### PR TITLE
fix: Improved installation section of Readme.md

### PR DESCRIPTION
**Pull request by vj-codes**
#### Is your pull request related to a problem? Please describe
 Improved installation section of Readme.md
Fixed issue #48

#### Checklist
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **[CONTRIBUTING](https://github.com/fbdevelopercircles/open-source-edu-bot/blob/master/CONTRIBUTING.md)** document.
- [ ] All new and existing tests passed.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,18 +26,18 @@ cd open-source-edu-bot
...
+On Windows
 
 ```PowerShell
 py -3 -m venv venv
 ```
 
 **Activate the environment**
 
-Copy the environments file and adapt it:
+Rename the environments file and adapt it:
```
Comment (elinguiuriel) [inline README.md:37]: Do not `rename` but `copy`, revert this change

File: README.md [hunk 2]
```diff
@@ -26,18 +26,18 @@ cd open-source-edu-bot
...
 
 ```bash
-cp .sample.env .env
+rename .sample.env .env
```
Comment (elinguiuriel) [inline README.md:40]: Not `rename` but `copy`

File: README.md (hunk 1 above)
Comment (aboullaite) [reply README.md:37]: I am the one who asked for the rename suggestion! Only copying is not clear for non-*unix users! @vj-codes I think renaming alone is also a bit misleading! so probably we combine both, copy and rename 👍

File: README.md (hunk 2 above)
Comment (aboullaite) [reply README.md:40]: @vj-codes this should not change! you could instead add 2 examples, one for *unix users (using `cp` cmd) and another one for windows! how does it sound ?

Comment (vj-codes) [reply README.md:40]: > @vj-codes this should not change! you could instead add 2 examples, one for *unix users (using `cp` cmd) and another one for windows! how does it sound ?

Great I'll change it! Sorry got confused:)

Comment (aboullaite) [reply README.md:40]: yes

File: README.md (hunk 1 above)
Comment (elinguiuriel) [reply README.md:37]: Actually the file `.sample.env` should not be `modified`, so not `renamed`. If the admin mess up with the config he can always restart by copying it.

File: README.md (hunk 2 above)
Comment (elinguiuriel) [reply README.md:40]: same remark as [here](https://github.com/fbdevelopercircles/open-source-edu-bot/pull/50#discussion_r460918707)!

**Comment by aboullaite** [comment]
Thanks @vj-codes for bearing with us for this PR! Amazing work
